e2e/test/scenarios/admin/databases.cy.spec.js

```diff
@@ -880,3 +897,14 @@ H.describeWithSnowplow("add database card", () => {
     });
   });
 });
+
+function editDatabase() {
+  cy.findByTestId("database-connection-info-section")
+    .findByRole("button", { name: "Edit" })
+    .click();
+}
+
+function forceTypeAndBlurUsingLabel(label, value) {
```

Comment:
I'm having a hard time getting the inputs to work as expected without `force: true`. Best I can tell is cypress tries to scroll the element into view but it's actually behind the header of the modal making it not clickable... I'm not sure what a better work around is yet.

Reply:
i think moving the scroll container element might help. I don't think we should merge all this with force - it's kind of invalidating any testing that the form is properly interactable